### PR TITLE
Bump scatter2d entry on regl-splom and fix for scattergl with multiple colors

### PR DESCRIPTION
**Pull request by archmoj**
As @dy noticed `regl-splom` uses wrong `regl-scatter2d` entry, pointing to not a bundle.
This branch tested with CI the changes of the [regl-splom PR](https://github.com/gl-vis/regl-splom/pull/10).
Also #3232 & https://github.com/plotly/phoenix-integration/issues/232 would be fixed in regards with [regl-scatter2d PR](https://github.com/gl-vis/regl-scatter2d/pull/15).
@etpinard

### CODE REVIEW

**Comment by etpinard** [comment]
Ok. Can we test that and https://github.com/gl-vis/regl-scatter2d/pull/15 together?

**Comment by etpinard** [comment]
Why is this a regression?

**Comment by archmoj** [comment]
#3232 was a regression.

**Comment by etpinard** [comment]
Great. Tests are passing on my machine too!

**Comment by etpinard** [comment]
@archmoj this seems like the appropriate base branch to get https://github.com/gl-vis/regl-splom/pull/10 and https://github.com/gl-vis/regl-scatter2d/pull/15 into plotly.js.

Would you mind (1) adding a test mock that locks down https://github.com/plotly/plotly.js/issues/3232 (2) close https://github.com/plotly/plotly.js/pull/3315 and https://github.com/plotly/plotly.js/pull/3242

**Comment by archmoj** [comment]
OK.

**Comment by archmoj** [comment]
@etpinard a new mock as well as a jasmine test were added to lock #3232.

File: package-lock.json [hunk 1]
```diff
@@ -8780,31 +8860,54 @@
       }
     },
     "regl-scatter2d": {
-      "version": "3.0.6",
-      "resolved": "https://registry.npmjs.org/regl-scatter2d/-/regl-scatter2d-3.0.6.tgz",
-      "integrity": "sha512-l2/OcCRKTxsCtrGtb2TKUKYnDHzI07qOm2eK2kiRYKyDwiWiGyiLC6p3SlOxDoqhQ/8gbIue9BABPXuCJ0lpRQ==",
+      "version": "git://github.com/gl-vis/regl-scatter2d.git#4b5a08f5c18616055f3f630a1b1592f2a681da0f",
+      "from": "git://github.com/gl-vis/regl-scatter2d.git#4b5a08f5c18616055f3f630a1b1592f2a681da0f",
```
Comment (archmoj) [inline package-lock.json]: https://github.com/gl-vis/regl-scatter2d/pull/15

File: package-lock.json [hunk 2]
```diff
@@ -8780,31 +8860,54 @@
...
+          }
+        }
       }
     },
     "regl-splom": {
-      "version": "1.0.4",
-      "resolved": "https://registry.npmjs.org/regl-splom/-/regl-splom-1.0.4.tgz",
-      "integrity": "sha512-+iq/RJAJdHCp48wPbEGQ5qw29OXFVF/m7CzcuLZxwptjdkB/FHGKiMuyqclOSNQcEKFxQTvRRJMJJ6brd8VvrA==",
+      "version": "git://github.com/gl-vis/regl-splom.git#17ec1b340fd7db810f5efe20516e8fbf11331c61",
+      "from": "git://github.com/gl-vis/regl-splom.git#17ec1b340fd7db810f5efe20516e8fbf11331c61",
```
Comment (archmoj) [inline package-lock.json]: https://github.com/gl-vis/regl-splom/pull/10

**Comment by dy** [comment]
@archmoj you can use real npm versions, I've published the two packages.

**Comment by archmoj** [comment]
Many thanks @dy. Just applied the changes using release versions.

**Comment by etpinard** [comment]
:dancer:

**Pull request merged**